catalog/imaging/imaging-transformer-chipping/pom.xml

```diff
@@ -101,13 +160,71 @@
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <!-- Don't Export Anything -->
                         <Export-Package/>
+                        <Import-Package>
+                            !org.codice.alliance.catalog.core.api.impl.types,
+                            !sun.security.action,
+                            *
+                        </Import-Package>
                         <Embed-Dependency>
-                            catalog-core-api-impl,
-                            platform-util
+                            catalog-core-api-impl;groupId=org.codice.alliance.catalog.core;version=${project.version},
+                            catalog-core-api-impl;groupId=ddf.catalog.core;version${ddf.version},
+                            platform-util,
+                            imaging-service-impl,
+                            imaging-transformer-nitf,
+                            jai-imageio-core,
+                            jai-imageio-jpeg2000,
+                            thumbnailator,
+                            commons-lang3,
+                            codice-imaging-nitf-core,
+                            codice-imaging-nitf-render,
+                            codice-imaging-nitf-fluent-api,
+                            la4j
```

Comment:
i used some stuff from imaging-transformer-nitf, which dragged in a bunch. But I'll review it again and see if anything can be trimmed.